src/Resistor/ResistorInput.css

```diff
@@ -0,0 +1,3 @@
+.ResistorInput {
+    padding-left: 10px;
+  }
```

Comment:
I'm not sure where the weird indent is coming from, but the closing curly brace should be in the first column.